src/components/sponsors/sponsor-group.js

```diff
@@ -39,4 +39,9 @@ SponsorGroup.propTypes = {
 
 SponsorGroup.styles = StyleSheet.create({
   group: {display: 'flex'},
+  sectionDivider: {
+    borderTop: '1px solid #9A9A9A',
```

Comment:
Where'd you get this color? Could we maybe move it to [here](https://github.com/javascriptair/site/blob/7f60730b3bcafb2fab232384054c593e57216113/src/styles/variables.js)?